server/schema/user/user.mutation.js

```diff
@@ -146,9 +145,16 @@ module.exports = new GraphQLObjectType({
       resolve: setUserBan,
     },
     setUserRoles: {
-      type: UserType,
+      type: GraphQLJSON,
```

Comment:
Do not use GraphQLJSON as a return type. Create a new GRaphQL type if required but keep it specific.